src/errors/value_exception.rs

```diff
@@ -72,7 +72,7 @@ impl PydanticCustomError {
         }
     }
 
-    #[getter(type)]
+    #[getter(r#type)]
```

Comment:
I didn't see what forced this change from [pyo3 changelog]

Reply:
I believe it was the switch to syn 2.0 changing the way the proc macro parsed idents.